README.rst

```diff
@@ -1,148 +1,262 @@
-- gunicorn 19.1.1 or above
+- RabbitMQ Server 3.3.0 or above (http://www.rabbitmq.com/download.html)
 
 Installation
-============
+------------
 Download the component by executing the following instruction:
+::
 
-git clone git@github.com:telefonicaid/fiware-facts.git
+    git clone git@github.com:telefonicaid/fiware-facts.git
 
-Usage
-=====
+Note: we recommend you to download this component into this location:
+``/opt/policyManager``
 
-Execute command "gunicorn --check-config server.py" inside the folder where you downloaded fiware-facts
 
-Changelog
-=========
+Configuration file
+------------------
+The configuration used by the fiware-facts component is optionally read from the file
+``conf/fiware-facts.cfg``
 
-v1.5.0 (2015-07-28)
--------------------
+Default values are found in ``facts/config.py``.
 
-Fix
-~~~
+MYSQL cloto configuration must be filled before starting fiware-facts component, user and password are empty by default.
 
-- Fixing imprecision while parsing from string to a number. [Guillermo
-  Jimenez Prieto]
+Options that user could define:
+::
 
-- Updating build.sh file to build in jenkins. [Guillermo Jimenez Prieto]
+    [common]
+     brokerPort: 5000       # Port listening fiware-facts
+     clotoPort:  8000       # Port listening fiware-cloto
+     redisPort:  6379       # Port listening redis-server
+     redisHost:  localhost  # Address of redis-server
+     rabbitMQ:   localhost  # Address of RabbitMQ server
+     cloto:      127.0.0.1  # Address of fiware-cloto
 
-- Fixing context value. NGSI parameter is called value in the latest
-  release of NGSI. [Guillermo Jimenez Prieto]
+    [mysql]
+     host: localhost        # address of mysql that fiware-cloto is using
+     user:                  # mysql user
+     password:              # mysql password
 
-- Context value parameter is called value in the last specification of
-  NGSI. [Guillermo Jimenez Prieto]
+    [logger_root]
+     level: INFO            # Logging level
 
-v1.4.0 (2015-06-29)
--------------------
+Running fiware-facts
+====================
 
-Fix
-~~~
+Execute command "gunicorn --check-config server.py" inside the folder where you downloaded fiware-facts
 
-- Checking if window size is instance of integer or long. [Guillermo
-  Jimenez Prieto]
+API Overview
+============
 
-- Fixing tests about getting windowsizes from cloto component.
-  [Guillermo Jimenez Prieto]
+Servers will update their context. The context information contains the description of the CPU, Memory, Disk and
+Network usages.
+
+An example of this operation could be:
+
+::
+
+        curl --include \
+             --request POST \
+             --header "Content-Type: application/json" \
+             --data-binary "{"contextResponses": [
+                {
+                    "contextElement": {
+                        "attributes": [
+                            {
+                                "value": "6",
+                                "name": "users",
```

Comment:
In a previous PR, I think that we delete the reference to users here and added the reference to the network use, but here I do not see it.